package.json

```diff
@@ -23,17 +23,22 @@
   },
   "homepage": "https://github.com/Runnable/Filibuster",
   "dependencies": {
-    "pty.js": "0.2.4",
+    "pty.js": "0.2.6",
     "which": "1.0.5",
     "express": "4.6.1",
     "eson": "0.5.0",
     "primus": "2.4.0",
     "ws": "0.4.31",
-    "substream": "0.1.1",
+    "substream": "0.1.3",
     "commander": "2.2.0",
     "dotenv": "^0.4.0"
   },
   "devDependencies": {
     "lab": "3.2.1"
+  },
+  "engineStrict": true,
```

Comment:
TIL: `engineStrict` is going to be deprecated. just a heads up - I know `pty` isn't backwards compatible, so this makes sense, but just fyi... https://docs.npmjs.com/files/package.json#enginestrict